app/src/androidTest/java/com/example/concordia_campus_guide/ExampleInstrumentedTest.java

```diff
@@ -1,27 +1,27 @@
-package com.example.concordia_campus_guide;
-
-import android.content.Context;
-
-import androidx.test.platform.app.InstrumentationRegistry;
```

Comment:
Is there a story created to fix UI tests?

Reply:
Not yet. However, we should open an issue for it asap